slippage_accounting.md

```diff
@@ -0,0 +1,51 @@
+# Solver Slippage Accounting
+
+Slippage accounting is performed on a per settlement/transaction basis according to the following two primary
+components. The SQL source code can be found
+on [GitHub](https://github.com/cowprotocol/solver-rewards/blob/main/queries/dune_v2/period_slippage.sql)
+or [Dune Analytics](https://dune.com/queries/2421375)
+
+## 1. Batch-wise Token Imbalance
+
+The token balance sheet represents a classified ledger all incoming and outgoing token transfers relative to the
+settlement contract. Classification categories are `USER_{IN/OUT}`, `AMM_{IN/OUT}` and `PHANTOM_TRANSFER`
+
+### Transfer Type Classification
+
+In all cases `IN` represents settlement contract as `recipient` and `OUT` as `sender`!
+
+- `USER_{IN/OUT}` transfers are those emitted by the Settlement contract's Trade Event (with `USER_IN` adjusted for
+  fees).
+- `AMM_{IN/OUT}` classification is assigned to all on-chain transfers that are NOT user transfers
+- `PHANTOM_TRANSFER` represents token transfers associated with _internalized buffer trades_. In brief, these are
+  transfers related to AMM interactions that were "skipped" for the purpose of gas optimization in favour of using the
+  settlement contract's internal holdings. The process for including this "off-chain" data in rather involved so we
+  dedicate and entire section to it (see below).
+
+Note that `AMM_IN/OUT` also captures `WETH` token (un)wraps.
+
+### Phantom Transfers (aka Internalized Token Imbalance)
+
+As mentioned above, as a form of gas optimization, solvers may indicate whether certain AMM interactions may be "
+internalized" if the settlement contract has sufficient balance and would be willing to facilitate the trade with their
+own funds.
```

Comment:
nanonits: somewhat weird line break but 🤷

Reply:
Yea, this was my IDE formatting according to a specified line width. Not sure I like it either. I would prefer to put one sentence per line (but some of them are a bit long). Will resort back to one line per sentence.

Reply:
changed to one sentence per line.